packages/block-editor/src/components/block-list/block.js

```diff
@@ -307,7 +274,6 @@ function BlockListBlock( {
 			'is-focused': isFocusMode && ( isSelected || isAncestorOfSelectedBlock ),
 			'is-focus-mode': isFocusMode,
 			'has-child-selected': isAncestorOfSelectedBlock,
-			'has-toolbar-captured': hasAncestorCapturingToolbars,
```

Comment:
Removing this class since it was used only for one broken CSS rule. Additionally it's a lot of selectors running for each block.